### PR TITLE
Add new jobs for openshift/managed-cluster-validating-webhooks

### PR DESCRIPTION
**Pull request by cblecker**

### CODE REVIEW

**Comment by cblecker** [comment]
/assign @stevekuznetsov 
for approval

File: ci-operator/config/openshift/managed-cluster-validating-webhooks/openshift-managed-cluster-validating-webhooks-master.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+base_images:
+  base:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.12
```
Comment (rogbas) [inline ci-operator/config/openshift/managed-cluster-validating-webhooks/openshift-managed-cluster-validating-webhooks-master.yaml]: This is a python based component, I wonder if we should update the image stream here.

Comment (cblecker) [reply ci-operator/config/openshift/managed-cluster-validating-webhooks/openshift-managed-cluster-validating-webhooks-master.yaml]: This is used just for clone/build time dependencies: https://github.com/openshift/ci-tools/blob/master/CONFIGURATION.md#build_root

In this case, python is needed at runtime, which is why we install it into the UBI image (https://github.com/openshift/managed-cluster-validating-webhooks/pull/27). There isn't a specific python release image.

**Pull request merged**